src/L1VotePool.sol

```diff
@@ -9,7 +9,7 @@ abstract contract L1VotePool {
 
   event VoteCast(
     address indexed voter,
-    uint256 indexed proposalId,
+    uint256 proposalId,
```

Comment:
I don't think this has to be changed as the underlying Governor should have a Tally compatible event since this is on L1.

Reply:
Agreed, this doesn't really need to change, since this contract would never be read by an alternative tool But I guess there's no downside to aligning. Unless we think it'd be advantageous to index it, I'm fine with including the ch ange.

Reply:
let me leave the change in, then. It feels slightly less confusing to me to have the signatures matching.